packages/coreutils/src/time.ts

```diff
@@ -15,9 +27,13 @@ export namespace Time {
    * @returns A formatted date.
    */
   export function formatHuman(value: string | Date): string {
-    let time = moment(value).fromNow();
-    time = time === 'a few seconds ago' ? 'seconds ago' : time;
-    return time;
+    // We must convert any strings to Date objects as that is the type expected by formatDate
+    value = convertToDateObject(value);
+
+    return formatDistanceToNow(value, {
+      includeSeconds: false,
+      addSuffix: true
+    });
```

Comment:
Can you comment on how backwards compatible these functions in our API are? For users using our coreutils function, will they notice any difference between before and after this PR?

Reply:
I was looking at the tests and I should have the 'includeSeconds' flag set to `true` as that's more consistent with the current behaviour. There are slight differences where date-fns could result in an output showing "less than 20 seconds ago" compares to moment.js which will display "19 seconds ago", so the output is not exactly as before.

https://date-fns.org/v2.0.0/docs/formatDistanceToNow